README.md

```diff
@@ -27,6 +27,7 @@
 - `touch` whether it has a touch screen.
 - `firefoxOS` whether Firefox OS is supported.
 - `os` the OS type it usually ships with.
+- `preset` whether it is in the default enabled set of devices
```

Comment:
Nit: It's unclear what "default enabled set of devices" means here. Also needs a final ".".